src/pages/VestingClaim.tsx

```diff
@@ -0,0 +1,191 @@
+        if (isValidBene) {
+          const vestingContract = new EthersContract(
+            currentBeneficiary[0].args?.vestingContract,
+            INVESTOR_EMPLOYEE_VESTING_CONTRACT_ABI,
+            provider.getSigner(),
+          ) as Vesting
+          setVestingContract(vestingContract)
+          try {
+            const remainingAmount = commify(
+              formatBNToString(
+                await SDLContract.balanceOf(vestingContract.address),
+                18,
+                2,
+              ),
+            )
+            setRemainingAmount(remainingAmount)
+          } catch (err) {
+            console.error(err)
+            notifyCustomError({
+              ...(err as Error),
+              message: "Unable to get total pending amount",
+            })
+          }
+          try {
+            const claimableVestedAmount = commify(
+              formatBNToString(await vestingContract.vestedAmount(), 18, 2),
+            )
+            setClaimableVestedAmount(claimableVestedAmount)
+          } catch (err) {
+            console.error(err)
+            notifyCustomError({
+              ...(err as Error),
+              message: "Unable to get vested amount",
+            })
+          }
+        }
+      } catch (err) {
+        console.error(err)
+        notifyCustomError({
+          ...(err as Error),
+          message: "Unable to query for vesting contracts",
+        })
+      }
+    }
+    void fetchBeneficiaries()
+  }, [account, chainId, library, isValidBene])
+
+  const onClaimClick: () => void = async () => {
+    if (!vestingContract) return
+    try {
+      const txn = await vestingContract.release()
+      notifyHandler(txn?.hash, "claim")
+      await txn?.wait()
+      setClaimableVestedAmount("0.0")
+    } catch (err) {
+      console.error(err)
+      notifyCustomError({
+        ...(err as Error),
+        message: "Unable to claim vested tokens",
+      })
+    }
+  }
+
+  return (
+    <>
+      <TopMenu activeTab={t("n/a")} />
+      <div className={styles.container}>
+        <div className={styles.gradient}></div>
+        <div className={styles.logoWrapper}>
+          <div className={styles.logo}>
+            <img src={logo} />
+          </div>
+        </div>
+        <div className={styles.mainContent}>
+          {isValidBene ? (
+            <>
+              <div className={styles.tokenBalance}>
+                {claimableVestedAmount} of {remainingAmount} remaining tokens
+                {canAdd && (
+                  <img
+                    src={plusIcon}
+                    className={styles.plus}
+                    onClick={() => addToken()}
+                  />
+                )}
+              </div>
+              <div className={styles.tokenBalanceHelpText}>
+                {t("totalClaimableSDL")}
+              </div>
+              <div className={styles.tokenBalanceHelpText}>
+                Vesting Contract:{" "}
+                <a
```

Comment:
nit: hide the link if we don't have a vesting contract address

Reply:
The containing div won't render if the beneficiary isn't valid which won't render this div as a result.